### ISSUE TITLE
Can't load templates

### ISSUE THREAD

**Issue by scottpham** [nblackburn/vue-brunch#5]
Hi there, thanks for this plugin, it's really useful. Unfortunately I cannot get the single-file components to load with templates as described in the Vue.js docs. The code in `module.exports` runs but without a template. If I manually add a `template` property with an html string it will work, but it kind of defeats the point of the single-file template idea. Could you take a look at my config?

```
module.exports = {
  paths: {

  },
  plugins: {
    sass: {
      options: {
        // an array of paths that will become available to sass' import statement
        includePaths: [
          'bower_components/bootstrap-sass-official/assets/stylesheets/'
        ]
      }
    }
  },
  // brunch will inject and concatenate files from bower, vendor, npm,
  // and app/scripts, in that order. For more info: http://brunch.io/docs/config#-files-
  files: {
    javascripts: {
      joinTo: {
        // custom files go in app/scripts. Entry point is main.js
        'app.js': 'app/scripts/*.js*',
        'vendor.js': 'app/scripts/vendor/*.js|node_modules/**|bower_components/**'
      },
      order: {
        // if necessary, list vendor scripts here in the order they need to run
        before: 'bower_components/jquery/dist/jquery.js'
      }
    },
    stylesheets: {
      // put .scss or css in app/styles
      joinTo: 'app.css'
    },
    templates: {
      joinTo: {
        'app.js': 'app/components/*.vue'
      }
    }
  }
}
```

and my `package.json dependcies`:

```
  "devDependencies": {
    "browser-sync-brunch": "0.0.9",
    "brunch": "^2.8.2",
    "css-brunch": "^2.6.1",
    "javascript-brunch": "^2.0.0",
    "json-brunch": "^1.5.4",
    "sass-brunch": "^2.6.3",
    "vue-brunch": "^1.2.1"
}
```

**Comment by nblackburn** [comment]
What version of Vue are you using?

**Comment by scottpham** [comment]
1.0.26

**Comment by nblackburn** [comment]
OK, you will need to use a lower version as `1.2.1` is a experimental release for Vue 2.0 support but there is an issue with Brunch as it doesn't support converting process variables other than `NODE_ENV` which Vue 2.0 has started using.

https://github.com/brunch/brunch/issues/1510

I suggest trying `1.1.2` as that is the latest version for Vue 1.x.

**Comment by scottpham** [comment]
Hmm now I'm getting different errors in the terminal that I haven't seen before: 
```
- error: DEPS_RESOLVE_FAILED of app/templates/banner.vue failed. Co
uld not load module 'vue-hot-reload-api' from '/Users/scottpham/Dropbox/Code/wui/ap
p/templates'. Possible solution: add 'vue-hot-reload-api' to package.json and `npm
install`.
```

**Comment by nblackburn** [comment]
Try installing it as a peer dependency`npm install vue-hot-reload-api@2.0.5 --save-dev`.

**Comment by scottpham** [comment]
Ok that got closer but now, different errors! I've seen this one before but it's strange because I wasn't getting it before I installed the new dependencies:

```
warn: app/templates/banner.vue compiled, but not written. Check your javascripts.joinTo config
```

**Comment by nblackburn** [comment]
Try using the following in your config as it might be a little too strict, causing the error you are seeing...

```javascript
templates: {
    joinTo: 'scripts/app.js'
}
```

**Comment by scottpham** [comment]
hmmm, same error. I'm loading the component like so, from within `app/scripts/mainVue.js`:

```
.....
  components: {
      'my-banner': require('../components/banner.vue')
    },
.....
```
The path is correct as far as I can tell!

**Comment by nblackburn** [comment]
I imagine the path will be fine but Brunch won't be compiling the template because it's not in the whitelist of things to bundle.

**Comment by scottpham** [comment]
So I've managed to get it working. Thanks for your help! 

Turns out it works better if I put my components folder in my `javascripts` config:
```
    javascripts: {
      joinTo: {
        'app.js': 'app/scripts/*.*|app/components/*.*',
        'vendor.js': 'app/scripts/vendor/*.js|node_modules/**|bower_components/**'
      },
```
And just like that works like a charm. To test, I did try keeping this configuration and upgrading `brunch-vue` again and it didn't work, so I think the fix was both parts: downgrading my brunch plugin as you said and adjusting my config as above. 

Thanks for your help!
